content/en/docs/tasks/configure-pod-container/enforce-standards-webhook.md

```diff
@@ -0,0 +1,10 @@
+---
+title: Enforce Pod Security Standards with the Pod Security Webhook
+reviewers:
+- tallclair
+content_type: task
```

Comment:
In the front matter, also set `min-kubernetes-server-version`, and mention the feature gate in the Prerequisites section.